Source/init.cpp

```diff
@@ -3,8 +3,10 @@
  *
  * Implementation of routines for initializing the environment, disable screen saver, load MPQ.
  */
-#include "all.h"
+#include <string>

```

Comment:
why not add <string> to all.h?

Reply:
That's not good style IMO. It's hard to keep track of which includes are no longer used with a mother-of-all-includes header. Preferably only the headers that are used in a file should be included in that file.

Reply:
I'm actually working on cleaning up headers and reducing what is included with all.h